.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "6"
+  - "8"
```

Comment:
I bumped the node version because it was erroring out saying that **@typescript-eslint/eslint-plugin** required a later version.  If this is not acceptable, I can have a look at other options.

Reply:
I think node 8 is acceptable, vscode-jest also supports node >= 8. @firsttris will this node version bump ok for your repo?

Reply:
yes